### PR TITLE
Update balance sheet styling

### PR DESCRIPTION
**Pull request by leahtaylor**
Update balance sheet to match new prototype

Update balance sheet styling 
Update called up share capital not paid field styling
Update error message styling
Update save and continue button styling

SFA-336

### CODE REVIEW

**Comment by leahtaylor** [comment]
screen shot 2018-09-25 at 10 29 29

File: src/main/resources/templates/fragments/header.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
                     </div>
 
                      <div class="govuk-header__content">
-                        <a id="proposition-name" href="/" class="govuk-header__link govuk-header__link--service-name"> File company accounts </a>
+                        <a id="proposition-name" href="/" class="govuk-header__link govuk-header__link--service-name"> File your CIC report and accounts </a>
```
Comment (BSaunders95) [inline src/main/resources/templates/fragments/header.html]: Should we be adding CIC details yet since the application won't support CICs until much later?

Comment (leahtaylor) [reply src/main/resources/templates/fragments/header.html]: Just going by the prototype

File: src/main/resources/templates/fragments/numberedHeading.html [hunk 2]
```diff
@@ -5,8 +5,8 @@
         <title></title>
     </head>
     <body>
-        <h1 id="page-title" th:fragment="numberedHeading" class="heading-xlarge">
-            <span id="page-title-number" class="circle circle-step-large" th:text='${number}'></span>
+        <h1 id="page-title" th:fragment="numberedHeading" class="govuk-heading-l">
+            <!--<span id="page-title-number" class="circle circle-step-large" th:text='${number}'></span>-->
```
Comment (BSaunders95) [inline src/main/resources/templates/fragments/numberedHeading.html]: Should this be commented?

Comment (leahtaylor) [reply src/main/resources/templates/fragments/numberedHeading.html]: ive left this in as the testers are raising a query with rachel whether the circled numbers should be in the headings or not

File: src/main/resources/templates/fragments/header.html (hunk 1 above)
Comment (leahtaylor) [reply src/main/resources/templates/fragments/header.html]: just going by the prototype given in the story https://ch-accounts-prototype-dev.herokuapp.com/small-full/small-full-balance-sheet-mvp

**Comment by BSaunders95** [comment]
Can the `dropDownHelp` fragment be removed as part of this PR too please because it won't be used any more?

**Pull request merged**